content/en/profiler/profiler_troubleshooting.md

```diff
@@ -232,38 +232,125 @@ For further help with this issue, [contact support][2] and include the output of
 
 ## Missing profiles in the profile search page
 
-If you've configured the profiler and don't see profiles in the profile search page, here are a few settings to check:
+If you've configured the profiler and don't see profiles in the profile search page, here are a few settings to check depending on the Operating System.
+
+{{< tabs >}}
+
+{{% tab "Linux" %}}
+
+1. Check that the Agent is installed and running.
+
+2. Check the profiler has been loaded from the loader log:
+
+   1. Open the `dotnet-native-loader-dotnet-<pid>` log file in the `/var/log/datadog` folder.
+
+   2. Look for `CorProfiler::Initialize: Continuous Profiler initialized successfully.` near the end.
+
+   3. If not present, enable debug logs by setting the `DD_TRACE_DEBUG` environment variable for the application.
```

Comment:
By "If not present" do you mean, if that `initialized successfully` message is not present? If so, I think we should make this part of step 2, and change the first words of this sentence to "If that message is not present..."

Reply:
✅ done